card/mod/standard/set/all/rich_html/form.rb

```diff
@@ -125,8 +125,8 @@ def editor_in_multi_card
     end
   end
 
-  def multi_card_edit_fields
-    nested_fields_for_edit.map do |name, options|
+  def multi_card_edit fields_only=false
```

Comment:
#multi_card_edit handles all nests (fields or not) by default.  but the most prominent call is from edit_slot, which restricts to fields.